bundles/org.openhab.core.automation.module.script/src/main/java/org/openhab/core/automation/module/script/internal/handler/ScriptConditionHandler.java

```diff
@@ -33,9 +33,9 @@
  */
 public class ScriptConditionHandler extends AbstractScriptModuleHandler<Condition> implements ConditionHandler {
 
-    public final Logger logger = LoggerFactory.getLogger(ScriptConditionHandler.class);
+    public static final String TYPE_ID = "script.ScriptCondition";
 
-    public static final String SCRIPT_CONDITION = "script.ScriptCondition";
+    public final Logger logger = LoggerFactory.getLogger(ScriptConditionHandler.class);
```

Comment:
Hm, public loggers are a uncommon thing, but it has been present before, so no need to change anything at this time.

Reply:
Noted! I forsee a @Null annotation cleanup coming for automation, and can clean up some of the other small stuff at that time.

Reply:
Done